docs/contributing.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Contributor guide
+description: Contributor guide
+keywords: [Contributor, guide]
+---
+
+# Contributing to Casdoor
+
+Welcome to Casdoor! This document is a guideline about how to contribute to Casdoor.
+
+If you find something incorrect or missing, please leave comments / suggestions.
+
+## Get involved
+
+There are many ways to contribute to Casdoor, here are some ideas to get started:
+
+Use Casdoor and report issues! When using Casdoor, report issues to promote development of Casdoor, no matter bugs or 
+proposal. Before file an issue on GitHub, it would be better to discuss first on Gitter, Casbin Forum, Google group or 
```

Comment:
plz add gitter link, forum link, qq group link like help page:https://casdoor.org/help

Do we have a google group？
if we have, plz add link.